agent/handlers/utils/helpers.go

```diff
@@ -129,3 +131,10 @@ func LimitReachedHandler(auditLogger audit.AuditLogger) func(http.ResponseWriter
 		auditLogger.Log(logRequest, http.StatusTooManyRequests, "")
 	}
 }
+
+// WriteErrorToDefaultLogger write the requestType, httpStatusCode and responseJSON of error to the default logger
+func WriteErrorToDefaultLogger(httpStatusCode int, responseJSON []byte, requestType string) {
```

Comment:
nit: I think the function name is a bit misleading, there's no error passed as parameter, also the function is not reused anywhere.  I think I'd prefer to get rid of the function and just put the code directly in `WriteJSONToResponse`

Reply:
Thanks for the suggestion. function `WriteErrorToDefaultLogger` is now integrated into function `WriteJSONToResponse`.